haskell-src/Concordium/Types/Execution.hs

```diff
@@ -828,6 +841,14 @@ instance S.Serialize Event where
               TransferMemo {..} ->
                 S.putWord8 21 <>
                 S.put tmMemo
+              Interrupted {..} ->
```

Comment:
What magic constants are you referring to? Tags for serialization? How would having them as constants help instead of hinder?

Reply:
Yes the serialization tags. As I understand, they are part of the public API for the node. I think it would be better to treat it more like such instead of having them inline in the ser/deser- functions. 
I think it could be beneficial for developers integrating with the node and possibly also for future modifications to the transactions. 

This could either be via a document or const's with the appropriate documentation attached. 
But as I said, I don't necessarily think this should be part of this PR, also if we want it to stay like it is, then obviously it shouldn't.

Reply:
As discussed on slack, I don't believe this would improve maintainability, correctness, or readability.

I do think serialization of this should be documented, but in a separate document. Having constants for tags is not sufficient documentation to be useful.